charts/payment-integration/templates/NOTES.txt

```diff
@@ -0,0 +1,4 @@
+A payment service will run on all K8s cluster nodes.
+
+Since this template is intended to deploy one of Payone or PayPal Plus services -
```

Comment:
the content of `NOTES.txt` is shown after executing `helm upgrade|install` so this text doesn't help (better place would be README.md). I vote for using the default NOTEST.txt (and maybe adjust it to your needs, e.g. port etc.). You can get the default by creating a new default chart with `helm create my-chart`

Reply:
c9481ff